docs/source/guide/dataset_create.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Create a Dataset
+short: Create a Dataset
+date: 2023-08-16 11:52:38
+tier: enterprise
+order: 0
+order_enterprise: 405
+meta_title: Create a Dataset to use with data discovery in Label Studio Enterprise
+meta_description: How to create a Dataset in Label Studio Enterprise using Google Cloud, Azure, or AWS.
+hide_sidebar: true
+---
+
+!!! note
+    At this time, we only support building datasets from unstructured data, meaning that the data must be in individual files rather than a structured format such as CSV or JSON.
```

Comment:
can we say "a bucket of unstructured data"? I think it could be confusing to imply that our requirement is "unstructured-ness", rather than "the data format we currently support is a filesystem"